namer/namer.cc

```diff
@@ -768,6 +768,15 @@ class NameInserter {
             ctx.state.mangleRenameSymbol(oldSym, oldSym.data(ctx)->name);
         }
         auto sym = ctx.state.enterTypeMember(asgn->loc, onSymbol, typeName->cnst, variance);
+
+        // Ensure that every type member has a LambdaParam with bounds, but give
+        // both bounds as T.untyped. The reason for this is that the bounds will
+        // be fixed up in the resolver, but if the type is used out of order (as
+        // in test/testdata/todo/fixed_ordering.rb) `T.untyped` will be used for
```

Comment:
I didn't understand the details of the comment, but it seems to me that this effectively makes an order dependency bug into a silent bug, where something silently becomes T.untyped. Did I understand it right? Do we want this behavior?

Reply:
This is preserving the existing behavior.

Reply:
https://github.com/sorbet/sorbet/blob/88a8efcdfe0f2a0b21437de9e7c33f4e0fe0ade8/namer/namer.cc#L800-L802